src/scripts.js

```diff
@@ -17,42 +13,34 @@ const searchBar = document.getElementById('searchBar');
 const navMyRecipes = document.getElementById('navMyRecipes');
 const navUserInfo = document.getElementById('navUserInfo');
 const logo = document.getElementById('logo');
-
-const recipeRepo = new RecipeRepository(recipeData);
+let users;
+let ingredients;
+let recipes;
+let recipeRepo;
 let modalRecipe;
 let currentUser;
 let savedView = false;
+let currentView;
 
 //event listeners
-window.addEventListener('load', () => {
-  getRandomUser();
-  createRecipeCards(recipeRepo.recipes);
-})
-
-recipeSection.addEventListener('click', (event) => {
-  if(event.target.className !== "all-recipes"){
-    createRecipeModal(event);
-  }
-});
+recipeSection.addEventListener('click', createRecipeModal);
 modalSection.addEventListener('click', collapseRecipe);
 filterDropdown.addEventListener('click', filterRecipes);
 navMyRecipes.addEventListener('click', showSavedRecipes);
-
-// Let's clean this up to be a proper form submission..?
-searchBar.addEventListener('keypress', function (e) {
-  if (e.key === 'Enter') {
-    searchRecipes();
-  }
-});
-
 logo.addEventListener('click', goHome);
-
-
-
-
-
+searchBar.addEventListener('keyup', searchRecipes);
 
 //functions
+apiCalls().then(data => {
```

Comment:
Great work to the team for figuring this one out

Reply:
Great work on apiCalls(). Functioning perfectly!